examples/hf_serialization.py

```diff
@@ -120,19 +121,15 @@ def serialize_model(
         config_path = f"{dir_prefix}-config.json"
         if (not config_file_exists) or force:
             logger.info(f"Writing config to {config_path}")
-            with stream_io.open_stream(
-                config_path, "wb+", *s3_write_credentials
-            ) as f:
+            with _write_stream(config_path) as f:
                 config_dict = (
                     config.to_dict() if hasattr(config, "to_dict") else config
                 )
-                f.write(json.dumps(config_dict).encode("utf-8"))
+                f.write(json.dumps(config_dict, indent=2).encode("utf-8"))
```

Comment:
What's the purpose of this change?

Reply:
It was there before this PR. HuggingFace `config.json` files are typically indented.